### DISCUSSION TITLE
Add .NET 6 and VS 2022 compatibility

### DISCUSSION THREAD

**Discussion by SirioAstarot** [BornToBeRoot/NETworkManager#1247] [Ideas]
Add support for the new Microsoft Visual Studio 2022, along with the framework .NET 6.

**Comment by BornToBeRoot** [comment]
Hi @SkylarAstarot,

you can use VS 2022 with .NET 5.

Is there any reason why i should upgrade to .NET 6 yet? 
.NET 5 is still supported and .NET 6 is in preview. I think i will upgrade once it's released.

**Reply by BornToBeRoot** [reply]
@SkylarAstarot Current master is on .NET 6 and the first version is released!
